sos/plugins/logs.py

```diff
@@ -11,26 +11,16 @@
 from sos.plugins import Plugin, RedHatPlugin, DebianPlugin, UbuntuPlugin
 
 
-class Logs(Plugin):
+class Logs(Plugin, RedHatPlugin, DebianPlugin, UbuntuPlugin):
     """System logs"""
 
     plugin_name = "logs"
     profiles = ('system', 'hardware', 'storage')
 
     def setup(self):
-        self.add_copy_spec([
-            "/etc/syslog.conf",
-            "/etc/rsyslog.conf",
-            "/etc/rsyslog.d"
-        ])
-
-        self.add_copy_spec("/var/log/boot.log")
-        self.add_journal(boot="this", catalog=True)
-        self.add_journal(boot="last", catalog=True)
```

Comment:
These collections are very important to several of our support teams.

Reply:
These are included now.